src/style/query_utils.js

```diff
@@ -11,8 +12,15 @@ function getMaximumPaintValue(property: string, layer: StyleLayer, bucket: Bucke
     if (value.kind === 'constant') {
         return value.value;
     } else {
-        return (bucket: any).programConfigurations.get(layer.id)
-            .paintPropertyStatistics[property].max;
+        const binders = (bucket: any).programConfigurations.get(layer.id).binders;
+        for (const name in binders) {
```

Comment:
😱  Why isn't `binders` keyed on property name? I remember trying this before but running into some issue that I don't remember now.

Reply:
It's keyed on _attribute_ name rather than property name.

Reply:
From a surface look, I don't see any obvious reason why we couldn't key it by property name instead.